tank/core/run.py

```diff
@@ -85,6 +85,8 @@ def create(self):
         """
         Create instances for the cluster.
         """
+        check_file_rights(self._app.cloud_settings.provider_vars['pvt_key'], '600')
```

Comment:
not necessarily 0600. 0400 is also a good option. The important requirement is the last two zeros